### ISSUE TITLE
renaissance testsuite fails on Windows due to exception: scala.MatchError: getOSInfo

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/aqa-tests#2471]
https://ci.adoptopenjdk.net/job/Test_openjdk11_j9_extended.perf_x86-64_windows/13/consoleFull
```
14:19:46  21/04/05 21:19:45 ERROR main: Harness failed with exception: scala.MatchError: com.ibm.lang.management.internal.ExtendedOperatingSystemMXBeanImpl@5b1d5e1 (of class com.ibm.lang.management.internal.ExtendedOperatingSystemMXBeanImpl)
14:19:46  	at org.renaissance.harness.JsonWriter.getOsInfo(ResultWriter.scala:168)
14:19:46  	at org.renaissance.harness.JsonWriter.getEnvironment(ResultWriter.scala:152)
14:19:46  	at org.renaissance.harness.JsonWriter.store(ResultWriter.scala:349)
```
Looks to have been fixed in renaissance https://github.com/renaissance-benchmarks/renaissance/pull/237/files

**Comment by sophia-guo** [comment]
There is a PR to revert renaissance jar from version 11 to 9, which probably fix this issue? https://github.com/AdoptOpenJDK/openjdk-tests/pull/2468/files 

@patkarns did you see this issue with renaissance version 9?

**Comment by smlambert** [comment]
This should be temporarily addressed by falling back to version 0.9 in https://github.com/AdoptOpenJDK/openjdk-tests/pull/2468 which is now merged.  We will have to watch for what version that [fix](https://github.com/renaissance-benchmarks/renaissance/pull/237/files) lands in, and test whether we can move up to that version for all platforms without regressing.
